README.md

```diff
@@ -1,35 +1,143 @@
+    + [Usage](#usage)
+    + [Role Variables](#role-variables)
+    + [Example Playbook](#example-playbook)
+    + [Summon & Service Managers](#summon---service-managers)
+    + [Recommendations](#recommendations)
+  * [Conjur Ansible Lookup Plugin](#conjur-ansible-lookup-plugin)
+    + [Environment variables](#environment-variables)
+    + [Role Variables](#role-variables-1)
+    + [Example Playbook](#example-playbook-1)
+  * [Contributing](#contributing)
+  * [License](#license)
 
-### Requirements
+<!-- Table of contents generated with markdown-toc
+http://ecotrust-canada.github.io/markdown-toc/ -->
 
-- [CyberArk Conjur Open Source](https://www.conjur.org) v1.x+ or
-- [CyberArk Dynamic Access Provider](https://docs.cyberark.com/Product-Doc/OnlineHelp/AAM-DAP/Latest/en/Content/Resources/_TopNav/cc_Home.htm) v10.x+
-- Ansible v2.9+
+## Requirements
+
+- An instance of [CyberArk Conjur Open Source](https://www.conjur.org) v1.x+ or [CyberArk Dynamic
+  Access
+  Provider](https://docs.cyberark.com/Product-Doc/OnlineHelp/AAM-DAP/Latest/en/Content/Resources/_TopNav/cc_Home.htm)
+  v10.x+ accessible from the target node
+- Ansible >= 2.9
+
+## Installation 
+From terminal, run the following command:
+```sh
+ansible-galaxy collection install cyberark.conjur
+```
+
+## Conjur Ansible Role
+This Ansible role provides the ability to grant Conjur machine identity to a host. Based on that
+identity, secrets can then be retrieved securely using the [Conjur Lookup
+Plugin](#conjur_variable-lookup-plugin) or using the [Summon](https://github.com/cyberark/summon)
+tool (installed on hosts with identities created by this role).
+
+### Usage
+The Conjur role provides a method to establish the Conjur identity of a remote node with Ansible.
+The node can then be granted least-privilege access to retrieve the secrets it needs in a secure
+manner.
 
 ### Role Variables
 
-None.
-<br>
-<br>
+* `conjur_appliance_url` (_Optional)_: URL of the running Conjur service
+* `conjur_account` (_Optional)_: Conjur account name
+* `conjur_host_factory_token` (_Optional)_: [Host
+  Factory](https://developer.conjur.net/reference/services/host_factory/) token for layer
+  enrollment. This should be specified in the environment on the Ansible controlling host.
+* `conjur_host_name` (_Optional)_: Name of the host to be created.
+* `conjur_ssl_certificate`: Public SSL certificate of the Conjur endpoint
+* `conjur_validate_certs`: Boolean value to indicate if the Conjur endpoint should validate
+  certificates
+* `summon.version`: version of Summon to install. Default is `0.8.2`.
+* `summon_conjur.version`: version of Summon-Conjur provider to install. Default is `0.5.3`.
+
+The variables marked with _`(Optional)`_ are required fields. All other variables are required for
+running with an HTTPS Conjur endpoint.
+
+### Example Playbook
+
+Configure a remote node with a Conjur identity and Summon:
+```yml
+- hosts: servers
+  roles:
+    - role: cyberark.conjur-host-identity
```

Comment:
is this definition the same once it's part of the collection?

Reply:
I'm not sure, but I should be able to answer this once I've finished testing more

Reply:
might be good to know for sure before we merge, since this will land in the project README on merge

Reply:
@BradleyBoutcher I think this still needs to be verified ...

Reply:
Based on the documentations and my short experimenting, the correct syntax is `cyberark.conjur.conjur_host_identity` when using this role installed through the collection.